config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -686,6 +686,60 @@ periodics:
     testgrid-tab-name: cloud-provider-azure-master-vmss-capz
     testgrid-alert-email: kubernetes-provider-azure-oot@googlegroups.com
     description: "Runs Azure specific tests periodically with cloud-provider-azure (https://github.com/kubernetes-sigs/cloud-provider-azure) on vmss using a stable capz release."
+- cron: '0 18 * * *'  # Run at 18:00 UTC everyday
+  # cloud-provider-azure-vmssflex-capz runs Azure specific tests periodically on vmss flex.
+  name: cloud-provider-azure-vmssflex-capz
+  decorate: true
+  decoration_config:
+    timeout: 5h
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    preset-azure-cred-only: "true"
+    preset-azure-anonymous-pull: "true"
+  extra_refs:
+    - org: jackfrancis
+      repo: cluster-api-provider-azure
+      base_ref: vmss-flex-cloudprovider
+      path_alias: sigs.k8s.io/cluster-api-provider-azure
+      workdir: true
+    - org: kubernetes-sigs
+      repo: cloud-provider-azure
+      base_ref: master
+      path_alias: sigs.k8s.io/cloud-provider-azure
+      workdir: false
+  spec:
+    containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221020-96ba49123f-master
+        command:
+        - runner.sh
+        args:
+        - ./scripts/ci-entrypoint.sh
+        - bash
+        - -c
+        - >-
+          cd ${GOPATH}/src/sigs.k8s.io/cloud-provider-azure &&
+          make test-ccm-e2e
+        securityContext:
+          privileged: true
+        env:
+        - name: TEST_CCM
+          value: "true"
+        - name: CONTROL_PLANE_MACHINE_COUNT
+          value: "1"
+        - name: KUBERNETES_VERSION
+          value: "latest"
+        - name: CLUSTER_TEMPLATE
+          value: https://raw.githubusercontent.com/jackfrancis/cluster-api-provider-azure/vmss-flex-cloudprovider/templates/cluster-template-external-cloud-provider-machinepool.yaml
+        - name: AZURE_LOADBALANCER_SKU
+          value: "Standard"
+        - name: CLUSTER_PROVISIONING_TOOL
+          value: "capz"
```

Comment:
You didn't add any new env var for vmss flex in ccm e2e. Then I guess it runs smoothly? Maybe we can add one just in case we need it someday. e.g. `TEST_VMSS_FLEX`

Reply:
Yes, it should run smoothly. The CCM will automatically figure out the node type. The yaml template used by capz controls whether to use vmss flex vm as nodes.
I just added a TEST_VMSS_FLEX=true in case feature usage